docs/options.md

```diff
@@ -340,9 +341,9 @@ Doing so produces a large `git diff`, and if it goes unnoticed during code revie
 If you want to make sure that your git repository only contains Linux-style line endings in files covered by Prettier:
 
 1. Set `endOfLine` option to `lf`
-1. Configure [a pre-commit hook](precommit.md) that will run Prettier
-1. Configure Prettier to run in your CI pipeline using [`--check` flag](cli.md#check)
-1. Ask Windows users to run `git config core.autocrlf false` before working on your repo so that git did not convert `LF` to `CRLF` on checkout.
+2. Configure [a pre-commit hook](precommit.md) that will run Prettier
+3. Configure Prettier to run in your CI pipeline using [`--check` flag](cli.md#check)
+4. Ask Windows users to run `git config core.autocrlf false` before working on your repo so that git did not convert `LF` to `CRLF` on checkout.
```

Comment:
Can you change this back?

Reply:
> Can you change this back?

I didn't change it manually, maybe some other tools doing this, I'll revert it, thanx.